schema_salad/main.py

```diff
@@ -105,6 +105,21 @@ def main(argsl=None):  # type: (Optional[List[str]]) -> int
         help="Generate classes in target language, currently supported: python",
     )
 
+    parser.add_argument(
+        "--codegen-target",
+        type=str,
+        default=None,
+        help="Defaults to sys.stdout for python (and ./ for hidden Java codegen)",
+    )
+
+    parser.add_argument(
+        "--codegen-examples",
+        type=str,
+        metavar="directory",
+        default=None,
+        help="Directory of example documents for test case generation (Java only).",
```

Comment:
Is there an example of how to use this?

Reply:
https://github.com/common-workflow-language/schema_salad/blob/master/schema_salad/tests/test_java_codegen.py#L23

I wanted to allow valid and invalid document tests but I never got that far. Currently just assumes all the files in the examples directory are valid documents of this type.